src/algebra/order/to_interval_mod.lean

```diff
@@ -733,3 +733,78 @@ lemma to_Ico_mod_zero_one (x : α) : to_Ico_mod (0 : α) zero_lt_one x = int.fra
 by simp [to_Ico_mod_eq_add_fract_mul]
 
 end linear_ordered_field
+
+section union
+-- porting note: move these lemmas to `algebra.order.archimedean` when the port is complete
```

Comment:
`algebra.order.to_interval_mod` is not imported in `algebra.order.archimedean`, is it? So I don't think we don't need this porting note at all.

Reply:
Good point, thanks. I have addressed this at #18427